test/test_nn.py

```diff
@@ -10647,7 +10647,6 @@ def test_nll_loss_empty_tensor_reduction_none(self, device):
         self._nll_loss_helper([2, 3, 5, 7, 0], "none", torch.empty([2, 5, 7, 0], device=device), device)
 
     @unittest.skipIf(TEST_WITH_UBSAN, "division-by-zero error with UBSAN")
-    @expectedFailureXLA  # https://github.com/pytorch/xla/issues/1539
```

Comment:
Can you also delete the implementation of the expectedFailureXLA decorator because IIRC it's not used anywhere else?